doc/engine/Installing-the-OpenQuake-Engine-Nightly.md

```diff
@@ -34,28 +34,16 @@ sudo diff -urN /etc/openquake/openquake.cfg /etc/openquake/openquake.cfg.new_in_
 ```
 See an [example](openquake.cfg-diff-example.md).
 
-Same must be done with ```/usr/share/openquake/engine/celeryconfig.py``` and ```/usr/share/openquake/engine/celeryconfig.py.new_in_this_release```. Usually you can replace ```celeryconfig.py``` with ```celeryconfig.py.new_in_this_release```.
-
 Finally upgrade your database:
 
 ```bash
 oq-engine --upgrade-db
 ```
 
-**Please note**: PostgreSQL and RabbitMQ must be started, even on a worker node, to successful complete the upgrade. For example, the right upgrade procedure on a worker node is:
-```
-sudo service postgresql start
-sudo service rabbitmq-server start
-sudo apt-get install python-oq-.*
-## Optional, stop the unused services on a worker node
-sudo service postgresql stop
-sudo service rabbitmq-server stop
-```
-
-## Run OQ Engine, without calculation parallelization
+## Run OQ Engine
 You are now ready to run the OQ Engine. First, try running one of the demos included with the package. There are several demo calculations located in `/usr/share/openquake/risklib/demos`. Example:
 <pre>
-oq-engine --run-hazard=/usr/share/openquake/risklib/demos/hazard/SimpleFaultSourceClassicalPSHA/job.ini --no-distribute
+oq-engine --run-hazard=/usr/share/openquake/risklib/demos/hazard/SimpleFaultSourceClassicalPSHA/job.ini
```

Comment:
While ``--run-hazard`` works and it is not deprecated, I would recommend to use ``--run`` instead:

```bash
$ oq-engine --run-hazard=/usr/share/openquake/risklib/demos/hazard/SimpleFaultSourceClassicalPSHA/job.ini
```